### PR TITLE
TST: small change to csvs to test xml generation

### PR DESCRIPTION
**Pull request by tangkong**
Previous PR is working and (presumably) checking out the repo with an ssh key.  This aims to fully flex the xml-generation

### CODE REVIEW

**Comment by tangkong** [comment]
I don't think that specifying the ref in a workflow works for PRs originating from forks.  I also think we have to merge this to see if I'm right
image

**Comment by tangkong** [comment]
A bit more reading revealed that the `pull_request_target` workflow trigger gives the `GITHUB_TOKEN` write permissions, which we don't need here.  I'm going to try setting this to the standard `pull-request` trigger, which only grants read access.  

https://securitylab.github.com/resources/github-actions-preventing-pwn-requests/
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target

File: Spreadsheet/KFE/QRIX-alarms.csv [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ﻿#Indent,Branch,PV,Description,Latch,Delay,Filter,Guidance
-0,QRIX,,,,,,
\ No newline at end of file
+0,QRIX,,,,,,
+,,MR1K1:BEND:RTD:US:1_RBV,TEMPERATURE,,,,
```
[the remark this answers is not in the document]
Comment (tangkong) [reply Spreadsheet/KFE/QRIX-alarms.csv:3]: This PR wasn't meant to test the daff workflow at all, it was meant to test the xml generation with some modified repository permissions / push-access-bypass settings.

I just got side-tracked by the big red ❌  from the daff workflow

**Comment by tangkong** [comment]
If we want to fully restrict access, I think the github approved way of posting a comment is through the rest API through `github-script`: https://github.com/actions/github-script#comment-on-an-issue 

We can scope out a fine-grained access token specifically allowed to post comments only

**Comment by tangkong** [comment]
I'm going to request that we focus on the daff workflow in a different PR, since it's distracting from the goal of this effort.  

The diff workflow is close (I think)

**Comment by ZLLentz** [comment]
To finish here, the last step is probably to undo the tiny csv change, right?

**Comment by tangkong** [comment]
The csv change was to test xml push generation on PR merge, so it's actually necessary here.  The goal of this was to test if we could give this job permissions via deploy-key, rather than by just omitting branch protections

This was poorly established / lost in the nerd snipe 😆

**Comment by tangkong** [comment]
I'm going to merge so I can find out how I messed this one up and try again

**Comment by tangkong** [comment]
image
 ahahahahahaahahit worked

**Pull request merged**